### PR TITLE
[NO-ISSUE] feat: add note preview stage

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Added preview stage detail

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 1]
```diff
@@ -16,6 +16,8 @@ import Button from '~/components/Button.astro'
 
 O **Object Storage** é uma solução de armazenamento de dados escalável e flexível que armazena dados como objetos discretos, cada um contendo os próprios dados, metadados associados e um identificador único. O Object Storage oferece alta escalabilidade, acessibilidade e economia para diversas necessidades de armazenamento de dados não estruturados.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:17]:
```suggestion
O **Object Storage** é uma solução de armazenamento de dados escalável e flexível que armazena dados como objetos discretos, cada um contendo os próprios dados, metadados associados e um identificador único. Essa solução oferece alta escalabilidade, acessibilidade e economia para diversas necessidades de armazenamento de dados não estruturados.
```

**Review by gabriel-azion — APPROVED** [review]

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 2]
```diff
@@ -14,7 +14,9 @@ import Button from '~/components/Button.astro'
 
 <Badge variant="accent">Preview</Badge>
 
-O **Object Storage** é uma solução de armazenamento de dados escalável e flexível que armazena dados como objetos discretos, cada um contendo os próprios dados, metadados associados e um identificador único. O Object Storage oferece alta escalabilidade, acessibilidade e economia para diversas necessidades de armazenamento de dados não estruturados.
+O **Object Storage** é uma solução de armazenamento de dados escalável e flexível que armazena dados como objetos discretos, cada um contendo os próprios dados, metadados associados e um identificador único. Essa solução oferece alta escalabilidade, acessibilidade e economia para diversas necessidades de armazenamento de dados não estruturados.
+
+Object Storage está sendo lançado no estágio **Preview**.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:19]:
```suggestion
**Object Storage** está sendo lançado no estágio **Preview**.
```

**Review by hannah-sk — APPROVED** [review]

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx [hunk 3]
```diff
@@ -14,7 +14,9 @@ import Button from '~/components/Button.astro'
 
 <Badge variant="accent">Preview</Badge>
 
-**Object Storage** is a scalable and flexible data storage solution that stores data as discrete objects, each containing its own data, associated metadata, and a unique identifier. Object Storage offers high scalability, accessibility, and cost-effectiveness for various unstructured data storage needs.
+**Object Storage** is a scalable and flexible data storage solution that stores data as discrete objects, each containing its own data, associated metadata, and a unique identifier. This solution offers high scalability, accessibility, and cost-effectiveness for various unstructured data storage needs.
+
+**Object Storage** is now released in **Preview** stage.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx:19]:
```suggestion
The **Object Storage REST API** is now released in **Preview** stage.
```

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 4]
```diff
@@ -14,7 +14,9 @@ import Button from '~/components/Button.astro'
 
 <Badge variant="accent">Preview</Badge>
 
-O **Object Storage** é uma solução de armazenamento de dados escalável e flexível que armazena dados como objetos discretos, cada um contendo os próprios dados, metadados associados e um identificador único. O Object Storage oferece alta escalabilidade, acessibilidade e economia para diversas necessidades de armazenamento de dados não estruturados.
+O **Object Storage** é uma solução de armazenamento de dados escalável e flexível que armazena dados como objetos discretos, cada um contendo os próprios dados, metadados associados e um identificador único. Essa solução oferece alta escalabilidade, acessibilidade e economia para diversas necessidades de armazenamento de dados não estruturados.
+
+**Object Storage** está sendo lançado no estágio **Preview**.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:19]:
```suggestion
A **API REST de Object Storage** está sendo lançada no estágio **Preview**.
```

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 5]
```diff
@@ -10,13 +10,13 @@ import Button from '~/components/Button.astro'
 
 ## 30 de Janeiro, 2024
 
-### Object Storage
+### Object Storage REST API
 
 <Badge variant="accent">Preview</Badge>
 
-O **Object Storage** é uma solução de armazenamento de dados escalável e flexível que armazena dados como objetos discretos, cada um contendo os próprios dados, metadados associados e um identificador único. O Object Storage oferece alta escalabilidade, acessibilidade e economia para diversas necessidades de armazenamento de dados não estruturados.
+O **Object Storage** é uma solução de armazenamento de dados escalável e flexível que armazena dados como objetos discretos, cada um contendo os próprios dados, metadados associados e um identificador único. Essa solução Esta solução agora vem equipada com acesso à **Object Storage API REST** da Azion, proporcionando alta escalabilidade, acessibilidade e economia para diversas necessidades de armazenamento de dados não estruturados.
```
Comment (evandroazion) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:17]: Ficou duplicado "Essa solução" "Esta solução"

**Review by evandroazion — APPROVED** [review]

**Pull request merged**